src/userSync.js

```diff
@@ -0,0 +1,102 @@
+import * as utils from 'src/utils';
+
+const userSync = exports;
+// A queue of user syncs for each adapter
+const queue = {
+  image: []
+};
+
+/**
+ * @function fireSyncs
+ * @summary Trigger all user syncs in the queue
+ * @private
+ */
+function fireSyncs() {
+  try {
+    // Fire image pixels
+    queue.image.forEach((sync) => {
+      let bidderName = sync[0];
+      let trackingPixelUrl = sync[1];
+      utils.logMessage(`Invoking image pixel user sync for bidder: ${bidderName}`);
+      // insertAdjacentHTML expects HTML string - convert DOM object to string
+      let img = userSync.createImgObject(trackingPixelUrl);
+      if (img) {
+        utils.insertElement(img);
```

Comment:
I have not used `srcset` for triggering tracking pixels. How does it work? Each URL in the list gets fired? I will need to look into it.